### DISCUSSION TITLE
Announcement bar with "For updates follow @squidfunk on Twitter"

### DISCUSSION THREAD

**Discussion by mrdbourke** [squidfunk/mkdocs-material#4145] [Q&A] [answered]
Hello!

I'm using the Insiders version and installing and deploying from GitHub via GitHub Actions: https://github.com/mrdbourke/pytorch-deep-learning/blob/main/.github/workflows/make_docs.yml

Recently I've noticed my site having an announcement bar appears with the text: For updates follow @squidfunk on Twitter, though I didn't set this anywhere 🤔 .

Potentially this is a recent update?

If so, is it possible to remove/change it? (I'd like there to be none by default but I could also add my own if possible).

<img width="2032" alt="Screen Shot 2022-07-20 at 3 04 33 pm" src="https://user-images.githubusercontent.com/16750345/179901690-e4f9440c-8a88-4591-8b04-f50661458542.png">

**Comment by squidfunk** [comment, marked as answer]
Thanks for reporting! I'm confused, actually. I [added back the announcement bar in the `overrides` folder](https://github.com/squidfunk/mkdocs-material-insiders/blob/b6bdb0d3291b8b9710d53d750acab3aba27a8bfa/src/overrides/main.html#L36-L45) of the Insiders build, which is [not part of the official distribution](https://github.com/squidfunk/mkdocs-material-insiders/blob/b6bdb0d3291b8b9710d53d750acab3aba27a8bfa/MANIFEST.in#L3). If you clone the Insiders repository and use the same folder, the announcement bar would appear, but it doesn't appear that this is what you're doing. I'll need to manage somehow to reproduce this.

**Reply by squidfunk** [reply]
Okay, I think I have an idea of what _might be_ happening. You're not using the [recommended approach to install Insiders in CI](https://squidfunk.github.io/mkdocs-material/publishing-your-site/#__tabbed_1_2). You've probably copied it from [our workflow](https://github.com/squidfunk/mkdocs-material/blob/3547155aa9f8be30ad13410b82b7014531c59e73/.github/workflows/documentation.yml#L70-L73) from this repository, but you must know that we must deviate from the recommended approach due to some limitations of how the Python ecosystem works (we're building our project with our project 🌀).

As a test, I've tried redeploying the documentation with the latest Insiders version for [Isotopes](https://squidfunk.github.io/isotopes/), one of my other projects, with no announcement bar appearing whatsoever:

![screenshot-squidfunk-github-io-isotopes-1658302284514](https://user-images.githubusercontent.com/932156/179923435-44741075-662a-4d10-a8b9-d955a3fe32a6.png)

Could you switch to our recommended approach and check whether the issue persists?

**Reply by squidfunk** [reply]
I've added a note to our workflow:

https://github.com/squidfunk/mkdocs-material/blob/ebf02870fe5841e70f6937dad59838b899768c90/.github/workflows/documentation.yml#L70-L73

**Comment by mrdbourke** [comment]
This worked!

No more announcement bar! 

Thank you very much :)
